lahja/asyncio/endpoint.py

```diff
@@ -591,13 +665,13 @@ def stop(self) -> None:
         if config is not None and config.internal:
             # Internal events simply bypass going over the event bus and are
             # processed immediately.
-            self._process_item(item, config)
+            await self._process_item(item, config)
             return
 
         # Broadcast to every connected Endpoint that is allowed to receive the event
         compressed_item = self._compress_event(item)
         disconnected_endpoints = []
-        for name, remote in list(self._outbound_connections.items()):
+        for name, remote in list(self._full_connections.items()):
             # list() makes a copy, so changes to _outbount_connections don't cause errors
```

Comment:
This comment should be updated to say `_full_connections`. My careless typo probably foiled your search-replace :sweat_smile: